Core/Src/retro-go/gui.c

```diff
@@ -90,7 +90,12 @@ void gui_init_tab(tab_t *tab)
     // tab->status[0] = 0;
 
     sprintf(str_buffer, "Sel.%.11s", tab->name);
-    tab->listbox.cursor = odroid_settings_int32_get(str_buffer, 0);
+    // tab->listbox.cursor = odroid_settings_int32_get(str_buffer, 0);
+    tab_t *selected_tab = gui_get_tab(odroid_settings_MainMenuSelectedTab_get());
```

Comment:
Unfortunately `tab` doesn't know its index. That's the only workaround I've found.